dts/bindings/rtc/nxp,irtc.yaml

```diff
@@ -0,0 +1,33 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: IRTC
+
+compatible: "nxp,irtc"
+
+include: rtc.yaml
+
+properties:
+  reg:
+    required: true
+
+  clock-select:
+    type: int
+    required: true
+    description: |
+      The input clock select for IRTC.
+      0 <- 16.384 kHz
+      1 <- 32.768 kHz
```

Comment:
is this soc specific?

Reply:
Yeah, I believe so.

Reply:
is this selection happening as part of IRTC block or something in an SOC clock tree, if it's not part of IRTC it shouldn't be in this binding, and if it is, then the description should be generic and not soc specific